guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Hammer-cli.adoc

```diff
@@ -0,0 +1,54 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Hammer_CLI_{context}"]
+= Using IdM credentials to log in to the {Project} Hammer CLI
+
+This section describes how to log in to your {Project} Hammer CLI with your {FreeIPA} (IdM) login and password.
+
+.Prerequisites
+* You have enrolled your {ProjectServer} into IdM and configured it to use IdM for authentication.
+For more information, see {InstallingServerDocURL}Using_FreeIPA_{project-context}[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
+* An IdM server is running and reachable.
+* The host is configured to use IdM credentials to log users in to your {Project} Hammer CLI.
+For more information, see {InstallingServerDocURL}Configuring_the_Hammer_CLI_to_Use_FreeIPA_User_Authentication_{project-context}[Configuring the Hammer CLI to Use {FreeIPA} User Authentication] in _{InstallingServerDocTitle}_.
+* The host is an IdM client.
+
+.Procedure
+. Obtain a Kerberos ticket-granting ticket (TGT) on behalf of a {Project} user:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ kinit idm_user
+----
++
+[WARNING]
+====
+If, when you were setting {FreeIPA} to be the authentication provider, you enabled authentication for the {Project} API rather than the {Project} web UI, an attacker can now obtain an API session after the user receives the Kerberos TGT. The attack is possible even if the user did not previously enter the {Project} login credentials anywhere, for example in the browser.
```

Comment:
```suggestion
If, when you were setting {FreeIPA} to be the authentication provider, you enabled authentication for the {Project} API rather than the {ProjectWebUI}, an attacker can now obtain an API session after the user receives the Kerberos TGT.
The attack is possible even if the user did not previously enter the {Project} login credentials anywhere, for example in the browser.
```

From a stategic point of view, I think we should consider moving this to a section called "security considerations" or similar to explain and mention possible downsides. 

Any strong opinions? @mdolezelova